sonoff/my_user_config.h

```diff
@@ -299,6 +299,7 @@
 //  #define USE_SI1145                             // Enable SI1145/46/47 sensor (I2C address 0x60) (+1k code)
   #define USE_LM75AD                             // Enable LM75AD sensor (I2C addresses 0x48 - 0x4F) (+0k5 code)
 //  #define USE_APDS9960                           // Enable APDS9960 Proximity Sensor (I2C address 0x39). Disables SHT and VEML6070 (+4k7 code)
+  #define USE_MGC3130                              // Enable MGC3130 Electric Field Effect Sensor (I2C address 0x42)
```

Comment:
This should be commented out in order to be disabled by default and keep flash and memory as small as possible and let the users that have this sensor to enable it if they need it.

Reply:
Okay. I am not completely shure, if we get a problem, when we use a firmware with the enabled MGC3130-sensor, save this to the CFG and flash the same hardware later with a firmware without the enabled sensor.
I suppose the CONFIG will get scrambled somehow. Is this correct?

Reply:
No, because the code does not have any issue when it is modified. The scrambled issue happens only when changing the order in sonoff_settings.h file that uses the EEPROM part of the memory for CONFIG and SETTINGS.

Reply:
Okay, good to know.
I have commented it out now.